pkg/datapath/linux/config/config.go

```diff
@@ -543,34 +542,13 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 
 	if option.Config.EnableVTEP {
 		cDefinesMap["ENABLE_VTEP"] = "1"
-
-		l := len(option.Config.VtepEndpoints)
-		cDefinesMap["VTEP_NUMS"] = strconv.Itoa(l)
-
-		var (
-			ipb  strings.Builder
-			macb strings.Builder
-		)
-
-		ipb.WriteString("(__u32[]){ ")
-		macb.WriteString("(__u64[]){ ")
-
-		for i, ep := range option.Config.VtepEndpoints {
-			fmt.Fprintf(&ipb, "%#x, ", byteorder.NetIPv4ToHost32(ep))
-
-			mac := option.Config.VtepMACs[i]
-			vtep_mac, err := mac.Uint64()
-			if err != nil {
-				return err
-			}
-			fmt.Fprintf(&macb, "%#x, ", vtep_mac)
+		cDefinesMap["VTEP_MAP"] = vtep.Name
+		cDefinesMap["VTEP_MAP_SIZE"] = fmt.Sprintf("%d", vtep.MaxEntries)
+		if len(option.Config.VtepCIDRs) > 0 {
+			m := option.Config.VtepCIDRs[0].Mask
```

Comment:
It seems a bit odd to me to only respect the mask specified in the first VTEP CIDR, and apply those to all of the entries. Should the VTEP CIDR mask be an independent configuration parameter to make it clearer that it applies to all VTEP CIDRs in the configuration?

Reply:
I did this to write less and simple code :), you mean I could add an independent configuration parameter  `vtep-mask` like below ? 

```
  enable-vtep:   "true"
  vtep-endpoint: "10.169.72.34  10.169.72.34"
  vtep-cidr:     "10.0.66.0/30  10.1.1.0/30"
  vtep-mac:      "00:50:56:a0:7d:d8 00:50:56:a0:7d:d8"
  vtep-mask:    255.255.255.252
```

Reply:
I added vtep-mask option and pushed the changes :)